js/views/TestProfilePage.js

```diff
@@ -9,6 +9,7 @@ export default class extends BaseVw {
     this.options = options;
 
     this.listenTo(app.profile, 'change', this.render, this);
+    this.listenTo(app.profile.get('social'), 'update', this.render, this);
```

Comment:
There are some bugs on this page having to do with adding and removing social accounts where some unsaved form data is blown away. I don't want to spend too much time on an example page, so I'm letting those be for now. The main purpose of the page was to show an example of validation as well as working with a model with nested attributes.